ember.js/2021-04/Apr 16 2021.md

```diff
@@ -0,0 +1,34 @@
+# April 16, 2021
+
+## Agenda
+
+- (public) (5 min) [RFC: Deprecate `computed().meta()`](https://github.com/emberjs/rfcs/pull/441)→ FCP  (RJ / CG)
+- (public) (5 min) Deprecation volume leading to 4.0. Mitigation needed? (RJ)
+    - Should we move [emberjs/rfcs#674](https://github.com/emberjs/rfcs/blob/master/text/0674-deprecate-transition-methods-of-controller-and-route.md) (deprecate transition methods on route/controller) to `until: '5.0.0'`?
+- (public) (10 min) Does landing Embroider by default in `ember new foo` output require an RFC? (RJ / EF )
+    - Why / why not? (See [this comment](https://github.com/ember-cli/ember-cli/pull/9495#issuecomment-819081264) by simonihmig in the PR enabling Embroider by default with some concerns.)
+    - I also wanted to put this on the agenda. I may not be able to make the call today, if I'm not there I'm OK either way on this, it's mostly a process question, not an Embroider-specific question. If we need an RFC, I would appreciate a volunteer who is not me to write it. -EF
+
+## Attendees
+
+- [ ]  Yehuda Katz (YK)
+- [ ]  Tom Dale (TD)
+- [ ]  Kris Selden (KS)
+- [ ]  Leah Silber (LS)
+- [ ]  Robert Jackson (RJ)
+- [ ]  Igor Terzic (IT)
+- [ ]  Matthew Beale (MB)
+- [ ]  Edward Faulkner (EF)
+- [ ]  Dan Gebhardt (DG)
+- [ ]  Godfrey Chan (GC)
+- [ ]  Ricardo Mendes (RM)
+- [ ]  Chad Hietala (CH)
+- [ ]  Katie Gengler (KG)
+- [ ]  Melanie Sumner (MS)
+- [ ]  Jen Weber (JW)
+- [ ]  Chris Garrett (CG)
+
+## Action Items
```

Comment:
@kategengler were there any notes from this meeting that are public? the answers to the questions about deprecation volume and Embroider being a default are directly applicable on my end for various apps / clients, so I'm curious about what was decided there

Reply:
Hi! We're not the best at taking notes (we are _trying_ to at least record action items), but as you can see, even the agenda is useful.

My recollection is the end result was that we will have an RFC for enabling Embroider as default, and that some of the deprecations will be pushed to be `5.0`